content/docs/apps/static-egress.md

```diff
@@ -15,8 +15,8 @@ You can set up your cloud.gov application to communicate with any external servi
 
 Authentication options:
 
-* **cloud.gov client application connecting to external services:** If your application on cloud.gov needs to be a client of an existing server application outside of cloud.gov, your application on cloud.gov can have a TLS certificate that the server application validates, or you can use application-level authentication (such as Authorization headers). In addition, you can configure outside services to accept connections from the [cloud.gov egress address ranges]((#cloud-gov-egress-ranges). You can accept HTTPS and TCP connections.
-* **cloud.gov server application accepting external client connections:** If you have a server application on cloud.gov that needs to secure connections from a client application outside of cloud.gov, your server application can limit inbound IP ranges using a [user-provided service instance](https://docs.cloudfoundry.org/devguide/services/user-provided.html) of a [route service](https://docs.cloudfoundry.org/devguide/services/route-binding.html), and you can set up application-level client authentication, such as Authorization headers. (*Limitations:* In the current cloud.gov architecture, your server application on cloud.gov cannot use protocol-level authentication, such as a TLS certificate, to authenticate with a client application outside of cloud.gov. Your server application can only provide HTTPS services, not TCP services.)
+* **cloud.gov client application connecting to external services:** If your application on cloud.gov needs to be a client of an existing server application outside of cloud.gov, your application on cloud.gov can have a TLS certificate that the server application validates, or you can use application-level authentication (such as Authorization headers). In addition, you can configure outside services to accept connections from the [cloud.gov egress address ranges]((#cloud-gov-egress-ranges). There are no restrictions on outbound connections.
+* **cloud.gov server application accepting external client connections:** If you have a server application on cloud.gov that needs to secure connections from a client application outside of cloud.gov, your server application can limit inbound IP ranges using a [user-provided service instance](https://docs.cloudfoundry.org/devguide/services/user-provided.html) of a [route service](https://docs.cloudfoundry.org/devguide/services/route-binding.html), and you can set up application-level client authentication, such as Authorization headers. (*Limitations:* In the current cloud.gov architecture, your server application on cloud.gov cannot use protocol-level authentication, such as a TLS certificate, to authenticate clients connecting to a cloud.gov server application. Your cloud.gov server application can only serve HTTPS requests, not other TCP services.)
```

Comment:
In the sentence "In the current cloud.gov architecture, your server application on cloud.gov cannot use protocol-level authentication, such as a TLS certificate, to authenticate clients connecting to a cloud.gov server application." we now have both "your server application on cloud.gov" and "a cloud.gov server application" - we should use the same phrasing for both since they both refer to the same thing.